test/static_arp.ml

```diff
@@ -46,7 +46,7 @@ module Make(E : Mirage_protocols_lwt.ETHIF)(Clock : Mirage_clock_lwt.MCLOCK) (Ti
   let input t buffer =
     (* disregard responses, but reply to queries *)
     try
-    match Arpv4_wire.get_arp_op buffer with
+    match Cstruct.BE.get_uint16 buffer 6 with
```

Comment:
Why drop the Arp getters here?

Reply:
because there are none (see https://github.com/mirage/arp/blob/master/src/arp_packet.mli). arp does not use ppx_cstruct atm.

Reply:
doesn't need cstruct, but just exposing a manual get function might be nice in a future rev. (so no hardcoded integer in tcpip)

Reply:
would be great if you could open an issue on the arp repository with these enhancements, so we don't forget about that -- another possibility would be to provide random generators of arp frames into the arp package instead of here

Reply:
done in https://github.com/mirage/arp/issues/12